opennms-provision/opennms-provisiond/src/main/java/org/opennms/netmgt/provision/service/Provisioner.java

```diff
@@ -292,7 +293,7 @@ public NewSuspectScan createNewSuspectScan(InetAddress ipAddress, String foreign
     /**
      * <p>createForceRescanScan</p>
      *
-     * @param ipAddress a {@link java.net.InetAddress} object.
+     * @param nodeId
```

Comment:
I'd opt for removing the complete javadoc if you have to modify it, otherwise leave it as before.